03_Ansible_Kubernetes_Modules/README.md

```diff
@@ -11,23 +11,6 @@ biggest benefits of using Ansible in conjunction with existing Kubernetes
 resource files is the ability to use Jinja templating so that you can customize
 deployments with the simplicity of a few variables in Ansible.
 
-The easiest way to get started is to install the modules on your local machine
```

Comment:
I figure it's better to address this just once in the Setup section.

Reply:
+1